rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeper.java

```diff
@@ -658,6 +658,7 @@ public FileInfo load(String key) throws Exception
       avail += new File(CacheUtil.getDirPath(d, conf)).getUsableSpace();
     }
     avail = BYTES.toMB(avail);
+    avail -= DiskUtils.getCacheSizeMB(conf);
```

Comment:
I see DiskUtils.getCacheSizeMB returns -1 when it is unable to calculate size of a directory, in that case, avail will become greater than what was usable?